spdx/document.py

```diff
@@ -17,6 +17,15 @@
 from spdx.version import Version
 
 
+def _add_parens(required, text):
+    """
+    Add parens around a license expression if `required` is True, otherwise
+    return `text` unmodified.
+    """
+    if required:
+        return '(' + str(text) + ')'
```

Comment:
I wonder if using interpolation or format would not be nicer?
e.g. either:
- `return '(%s)' % (text)`
- `return '({text})'.format(a=text)`
and then maybe this simpler form (though the if construct is fine too):
- `return required and '({text})'.format(a=text) or text`